src/core/build_cache.jl

```diff
@@ -1,5 +1,9 @@
 function build_cache!(cache::TimeStatusChange, op_model::OperationModel)
-    parameter = get_value(op_model.canonical, cache.ref)
+    build_cache!(cache, op_model.canonical)
+end
+
+function build_cache!(cache::TimeStatusChange, canonical::CanonicalModel)
```

Comment:
Would it be better to put these functions in the file that defines TimeStatusChange (hopefully time_status_change.jl).

Reply:
This would be changed later when we do a file structure refactor.